capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -17,6 +21,15 @@ export class WikiMediaComponent implements OnChanges, OnInit {
   loading = true;
   body: string;
   urls = new Map();
+  langlinks = new Map();
+  languages: any = Languages as any;
+  language: any = Languages as any;
```

Comment:
Wouldn't this set both language and languages to the same object? Additionally, can we get more specific than " as any"?